dll/shellext/zipfldr/CZipCreator.cpp

```diff
@@ -315,6 +312,8 @@ unsigned CZipCreatorImpl::JustDoIt()
                                       Z_DEFAULT_STRATEGY,
                                       password,
                                       crc,
+                                      MINIZIP_COMPATIBLE_VERSION,
+                                      (bUtf8 ? MINIZIP_UTF8_FLAG : 0),
```

Comment:
Seems here that bUtf8 would always be TRUE (first because you declared it as "const", and because DoGetNameInZip doesn't modify bUtf8 as reference...)

Reply:
In the previous PR, some reviewer wanted to use non-UTF-8 zip. Just a memorial reason.